Apps/pholetools/oletools.json

```diff
@@ -0,0 +1,167 @@
+{
+    "appid": "6549eb4a-d15c-4e01-8e15-91b85dd77087",
+    "name": "oletools",
```

Comment:
Capitalize?

Reply:
I guess, no. But @dfederschmidt, could you please confirm this?

Reply:
`**Oletools**` works fine.